### PR TITLE
IJMP-1630 Added initial processing of the global Zowe team conf file

### PR DESCRIPTION
**Pull request by ktsytsen**

### CODE REVIEW

**Review by ATsikhamirau — APPROVED** [review]

File: src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt [hunk 1]
```diff
@@ -57,21 +67,42 @@ class UpdateZoweConfigAction : DumbAwareAction() {
       return
     }
     val vFile = e.getData(CommonDataKeys.VIRTUAL_FILE)
-    if (vFile?.path != "${project.basePath}/$ZOWE_CONFIG_NAME") {
+    var type = ZoweConfigType.GLOBAL
+    if (vFile?.path != ZoweConfigServiceImpl.getZoweConfigLocation(
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt:71]: if (
  vFile?.path != ZoweConfigServiceImpl.getZoweConfigLocation(project, ZoweConfigType.LOCAL) 
  && vFile?.path != ZoweConfigServiceImpl.getZoweConfigLocation(project, ZoweConfigType.GLOBAL)
)

File: src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt [hunk 2]
```diff
@@ -40,11 +41,20 @@ class UpdateZoweConfigAction : DumbAwareAction() {
       e.presentation.isEnabledAndVisible = false
       return
     }
+
+    var type = ZoweConfigType.GLOBAL
+    if (e.getData(CommonDataKeys.VIRTUAL_FILE)?.path == ZoweConfigServiceImpl.getZoweConfigLocation(
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt:46]: val zoweLocalConfigLocation = ZoweConfigServiceImpl.getZoweConfigLocation(project, ZoweConfigType.LOCAL)
if (e.getData(CommonDataKeys.VIRTUAL_FILE)?.path == zoweConfigLocalLocation)

File: src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt (hunk 1 above)
Comment (KUGDev) [reply src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt:71]: + better to use val zoweLocalConfigLocation and val zoweGlobalConfigLocation outside both actionPerformed and update functions

File: src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt [hunk 3]
```diff
@@ -57,21 +67,42 @@ class UpdateZoweConfigAction : DumbAwareAction() {
...
+        project,
+        ZoweConfigType.LOCAL
+      ) && vFile?.path != ZoweConfigServiceImpl.getZoweConfigLocation(
+        project,
+        ZoweConfigType.GLOBAL)
+    ) {
       e.presentation.isEnabledAndVisible = false
       return
     }
+    if (vFile.path == ZoweConfigServiceImpl.getZoweConfigLocation(project, ZoweConfigType.LOCAL))
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/explorer/zowe/actions/UpdateZoweConfigAction.kt:81]: same here

File: src/main/kotlin/org/zowe/explorer/zowe/service/ZoweConfigService.kt [hunk 4]
```diff
@@ -106,3 +111,12 @@ enum class ZoweConfigState(val value: String) {
     return value
   }
 }
+
+enum class ZoweConfigType(val value: String) {
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/explorer/zowe/service/ZoweConfigService.kt:115]: some docs, explaining the purpose of the class

**Review by KUGDev — APPROVED** [review]

**Review by dnlsv — APPROVED** [review]

**Pull request merged**